src/boot/z_std_dma.c

```diff
@@ -367,14 +486,23 @@ s32 DmaMgr_SendRequestImpl(DmaRequest* req, void* ram, uintptr_t vrom, u32 size,
     return 0;
 }
 
-s32 DmaMgr_SendRequest0(void* ram, uintptr_t vrom, u32 size) {
+/**
+ * Submit a synchronous DMA request. This will block the current thread until the requested transfer is complete. Data
+ * is immediately available once the current thread resumes execution.
```

Comment:
```suggestion
 * is immediately available when the function returns.
```
details about the thread resuming are technically correct but probably useless to the end user imo